src/model/data/fridge/REST.yaml

```diff
@@ -210,7 +239,7 @@ paths:
               schema:
                 type: object
                 properties:
-                  timestamp:
+                  editTimestamp:
```

Comment:
this is a bit confusing because it makes it sound like the Report was edited, but reports are not edited. They are added to the database table. I would suggest keeping it as timestamp

Reply:
Ok